src/typing_extensions.py

```diff
@@ -535,46 +495,6 @@ def _get_protocol_attrs(cls):
     return attrs
 
 
-def _maybe_adjust_parameters(cls):
```

Comment:
This function used to be used for our implementation of `TypedDict` and `Protocol`, but it is now only used for our implementation of `Protocol` on Python <3.8